classes/ValidFormBuilder/Password.php

```diff
@@ -86,22 +85,36 @@ public function __toHtml($submitted = false, $blnSimpleLayout = false, $blnLabel
 
         $varValue = $this->__getValue($submitted, $intCount);
 
-        $blnError = ($submitted && ! $this->__validator->validate($intCount) && $blnDisplayErrors) ? true : false;
+        $blnError = ($submitted && !$this->__validator->validate($intCount) && $blnDisplayErrors) ? true : false;
 
-        if (! $blnSimpleLayout) {
+        if (!$blnSimpleLayout) {
             // *** We asume that all dynamic fields greater than 0 are never required.
             if ($this->__validator->getRequired() && $intCount == 0) {
                 $this->setMeta("class", "vf__required");
             } else {
                 $this->setMeta("class", "vf__optional");
             }
 
+            if ($this->isRemovable()) {
+                $this->setMeta("class", "vf__removable");
+            }
+
+            //*** Add data-dynamic="original" or data-dynamic="clone" attributes to dynamic fields
```

Comment:
Please explain in short why you add the data attributes. There is no direct logic to derive the "why" from.